src/main/resources/static/api.yml

```diff
@@ -1823,3 +2006,93 @@ components:
         - type
         - subType
         - note
+    Assessment:
+      type: object
+      properties:
+        id:
+          type: string
+          format: uuid
+        application:
+            $ref: '#/components/schemas/Application'
+        assignedToProbationOfficer:
+          type: string
+          format: uuid
+        schemaVersion:
+          type: string
+          format: uuid
+        outdatedSchema:
+          type: boolean
+        createdAt:
+          type: string
+          format: date-time
+        assignedAt:
```

Comment:
`allocatedAt` would conform better perhaps to our terminology? Do we need "assign" at all?